Web_Development/Javascript/Topics/1.Hello_World/README.md

```diff
@@ -1 +1,110 @@
-# To Do
\ No newline at end of file
+# Hello World Tutorial In JavaScript
+In this tutorial you will learn how to print "Hello, World!" in JavaScript in 4 different ways.
+
+# Description
+A "Hello, World!" program is a basic program that displays the message "Hello, World!" on the screen. Because it's such a basic program, it's frequently used to teach novices a new programming language.
+
+The Following Ways to print Hello World on the screen are:
+
+## 1. Using console.log()
+
+```console.log()``` is mainly used when we are debugging JavaScript code. 
+
+__Source Code:__
+```javascript
+console.log('Hello, World!');
+```
+__Output:__
+```
+Hello, World!
+```
+__Explanation:__
+This Syntax prints``` Hello, World!```  string to the console.
+
+## 2. Using alert()
+The ```alert()``` method displays an alert box over the current tab/window with a specified message.
+
+Also in this you will learn how to add JavaScript in an HTML File and where you can see your output.
+
+__Source Code:__
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World Example</title>
+</head>
+<body>
+  This an example to add JavaScript in an HTML File.  
+</body>
+<script>
+    alert('Hello, World!');
+</script>
+</html>
+```
+__Explanation:__
+
+The Output will be generated in an alert box of your browser. 
+
```

Comment:
Here we can add a screenshot of the alert box of the browser with `Hello World!` printed inside it.